src/engine/Preprocessor.cpp

```diff
@@ -336,7 +339,8 @@ bool Preprocessor::processEquations()
 
                 lowerBound /= -ci;
 
-                if ( FloatUtils::gt( lowerBound, _preprocessed.getLowerBound( xi ) ) )
+                double diff = lowerBound - _preprocessed.getLowerBound( xi );
+                if ( FloatUtils::isPositive( diff, epsilon ) )
```

Comment:
Switch back to gt( x, y, epsilon ) to make it more readable

Reply:
Done